packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.component.js

```diff
@@ -44,11 +47,13 @@ class InputDateTimePicker extends React.Component {
 		onBlur: PropTypes.func,
 		dateFormat: PropTypes.string,
 		useSeconds: PropTypes.bool,
+		useTime: PropTypes.bool,
 	};
 
 	static defaultProps = {
 		dateFormat: 'YYYY-MM-DD',
 		useSeconds: false,
+		useTime: true,
```

Comment:
We should not have time by default.
So the classical use is only date, and when we want time, we just write `<InputDateTimePicker useTime>`.
(And I think we should rename the component to `InputDatePicker`)

Reply:
i guess if that component exist its already used somewhere so renaming it my lead to breaking changes ?

Reply:
No it's not used, as it's still in progress. It's a breaking changes but as it `should` not be used (warning in constructor) and it is not used, it's ok to do this breaking change.
But no pb to do that in another PR